environments.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
please add inline documentation to each of these new files what they are about and how they are used